operations/operations/company-policies/community-response-policy.md

```diff
@@ -0,0 +1,69 @@
+
+1 - Non-regulatory **Privacy/Telemetry** where opinions are at play more than legal or regulatory issues 
+* Approved Responders: Head of Community, Head of PM, CPO, CEO
+* Loop in: CEO
+* Template Response: To be added
+
+2 - Product-related **Regulations/GDPR**, and other legal issues
+* Approved Responders: Head of Community, Head of PM, Head of Legal, CPO, CEO
+* Loop in: Head of Legal, Head of PM
+* Template Response: To be added
+
+3 - **Licensing**, including open source and commercial license inquiries
+* Approved Responders: Head of Community, Head of PM, CEO
+* Template Response: To be added
+
+4 - **Packaging**, including requests for making an Enterprise feature free
+* Approved Responders: Head of PM, CPO
+* Template Response: To be added
+
+5 - **Workplace policies**, including hiring locations
+* Approved Responders: Director of HR, Head of Legal
+* Template Response: To be added
+
+6 - **Security vulnerabilities**
+* Approved Responders: Head of Community, Head of Security, Security PM, VP Eng
+* Loop in:  Head of Security, Security PM, VP Eng
+* Template Response: To be added
+
+7 - **Illicit use**
+* Approved Responders: Head of Community, Head of Marketing, CEO
+* Loop in: Head of Legal, Head of Marketing
+* Template Response: To be added
+
+8 - Marketing-related **Regulations/GDPR**, and other legal issues
+* Approved Responders: Head of Community, Head of Marketing, Head of Legal, CEO
+* Loop in: Head of Marketing, Head of Legal 
+* Template Response: To be added
+
+9 - **Governmental data request**, e.g. US government requesting user/customer data under US jurisdiction
+* Approved Responders: Head of Legal, CEO
+* Loop in: Head of Legal 
+* Template Response: To be added
+
+### Non-sensitive topics
+
+Non-sensitive topics include feature requests, troubleshooting questions, bug reports, and other user feedback. 
+
+Anyone in the company is welcome and encouraged to respond, regardless of the medium. If it is your first time responding to the community, please read [principles, tips and sample responses](https://docs.mattermost.com/process/community-guidelines.html#mattermost-community-forums) for help on various inquiries.
+
+### Non-compliance policy
+
+In cases where a staff member who is not on the approved responder list engages with a sensitive topic, one of the approved responders will contact the member in Mattermost to remind them of the community response policy, and ask if they would be interested in becoming an approved responder for one or more of the topics.
+
+Please note that we do not want to discourage you from engaging with the community! If you ever have questions about whether a topic is considered sensitive, ask in the “Community Escalations” channel or ask 1-1 from the Head of Community.
```

Comment:
Should we indicate who the head of community is, or direct to a page with a list?